### PR TITLE
Clean up Copyright statements

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

**Review by tsteenbe — CHANGES_REQUESTED** [review]
We can not touch the copyrights statements of third parties as their licenses require us to reproduce their exact copyright statement - ORT project itself has similar requirement in its Apache-2.0 license - https://github.com/oss-review-toolkit/ort/blob/main/LICENSE#L100-L104.

File: .reuse/dep5 [hunk 1]
```diff
@@ -64,22 +64,10 @@ Files: examples/*
...
-Files: integrations/jenkins/*
-Copyright: 2022 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
-License: Apache-2.0
-
-Files: integrations/schemas/*
+Files: integrations/*
 Copyright: 2021 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
 License: Apache-2.0
 
-Files: integrations/tekton/*
```
Comment (tsteenbe) [inline .reuse/dep5:87]: The Google attribution is already in the NOTICE file but this is the machine readable version so I would not remove it.

**Comment by sschuberth** [comment]
> We can not touch the copyrights statements of third parties as their licenses require us to reproduce their exact copyright statement

Which means we need to e.g. [update these lines](https://github.com/oss-review-toolkit/ort/pull/6837/commits/4c59787d88bf39c95a72fe48c9d3ddb2ba3257fd#diff-1c645862be9de205dccc305cde10248abfe5b5a2297fc0502cbdd22cae5c9305R97-R99) whenever the Gradle wrapper changes (in a new year), right?

**Comment by sschuberth** [comment]
I've dropped the last two commits.

**Comment by sschuberth** [comment]
> I've dropped the last two commits.

@tsteenbe please have another look.

**Comment by sschuberth** [comment]
> > I've dropped the last two commits.
> 
> @tsteenbe please have another look.

Ping once more @tsteenbe.

**Review by tsteenbe — APPROVED** [review]

**Comment by sschuberth** [comment]
Merging despite the unrelated `ConanFunTest` failure.

**Pull request merged**